### DISCUSSION TITLE
Does it make sense that your introductory video leads to a Youtube video that requires sign on?

### DISCUSSION THREAD

**Discussion by vonHabsi** [freenet/freenet-core#1290] [Q&A] [answered]
Does it make sense that your introductory video leads to a Youtube video that requires sign on?

Shouldn't you be dogfooding your stuff early on?

![Freenet Youtube Block 01](https://github.com/user-attachments/assets/dbc54775-3916-4596-99db-3fdb7beaca96)

**Comment by darkwiz666** [comment]
> Shouldn't you be dogfooding your stuff early on?

You're using "Dogfooding" as if Freenet has their own video hosting platform: they don't (AND you would still only be able to see the video with an out-proxy that had embeds that Github allowed [the project site is hosted here]). 

Your gripe should be that they're hosting the video on Youtube, which as owned by Google does not honor user privacy, which is antithetical to their project ethos, and that there are other options that better closely relate to what they're doing.

**They should host on Odysee/LBRY or Peertube.** At the very least it's hosted on X (and Elon cares far more about your privacy than the old owners, although he could do much, much more) and Rumble (where I don't think they'd care until it because antithetical to their positions).

As for the sign-in part: that has nothing to do with the project. If you're accessing Youtube from a Tor route, it will require you to prove you're a human thanks to people using it as a proxy to increase view-count and upvotes...a totally separate problem. **And then it's not even all the time...**

Brave Tor-tab, UK IP (detected as VPN), **not logged in....**:
![image](https://github.com/user-attachments/assets/c5df9f6a-208f-4a11-a15a-f1cd8904580f)

**Comment by sanity** [comment, marked as answer]
We provide links to the same video on X and Rumble. Eventually we'll have YouTube alternatives on Freenet but in the meantime I think purity testing tools is counterproductive.

**Reply by ghost** [reply]
You went with "purity testing tools" for your response? Oof.
I have to be choosy about the communities that I want to join, which means being aware of the people in that community and the people the community is advertising to. The services that you've chosen, and the ones you've omitted, tell me that the community you are trying to build is not one that I would feel safe in.
So, I guess, thanks for letting many of us know we should stay away from Freenet.
